content/applications/sales/sales/invoicing/expense.rst

```diff
@@ -1,65 +1,225 @@
+Expenses application
+====================
 
-You should also activate the analytic accounts feature to link expenses
-to the sales order, to do so, go to :menuselection:`Invoicing -->
-Configuration --> Settings` and activate *Analytic Accounting*.
+To be able to reinvoice a customer for an expense, the *Expenses* application **must** be installed.
 
-Add expenses to your sales order
-================================
+To install the *Expenses* application, navigate to :menuselection:`Main Odoo Dashboard --> Apps`,
+and click :guilabel:`Install` on the *Expenses* application block. When clicked, Odoo installs the
+application, refreshes the page, and returns to the main Odoo dashboard.
 
-From the expense app, you or your consultant can create a new one, e.g.
-the hotel for the first week on the site of your customer.
+Add expenses to sales orders
+============================
 
-You can then enter a relevant description and select an existing product
-or create a new one from right there.
+To begin, have a sales order confirmed in the *Sales* app to which a reinvoiced expense can be
+added. Or, create a new sales order from scratch. To do that, navigate to the :menuselection:`Sales
+app --> New`. Doing so reveals a blank quotation form.
 
-.. image:: expense/expense01.png
+Then, add a :guilabel:`Customer`, and add a product to the :guilabel:`Order Lines` tab, by
+clicking :guilabel:`Add a product`. Next, select a product from the drop-down menu.
+
+Lastly, click :guilabel:`Confirm` to confirm the sales order.
+
+.. image:: expense/confirmed-sales-order.png
+   :align: center
+   :alt: This is how a confirmed sales order looks in the Odoo Sales application.
+
+With the sales order confirmed, it's time to create an expense.
+
+To do that, navigate to the *Expenses* application, by going to :menuselection:`Main Odoo Dashboard
+--> Expenses`.
+
+Then, from the *Expenses* dashboard, click :guilabel:`New` to reveal a blank expenses form.
+
+.. image:: expense/blank-expenses-form.png
+   :align: center
+   :alt: A blank expenses form in the Odoo Expenses application.
+
+On the expenses form, add a :guilabel:`Description` to easily reference the expense.
+
+Then, in the :guilabel:`Category` field, select one of the following options from the drop-down
+menu:
+
+- :guilabel:`Communication`: any form of communication related to a project/order.
+- :guilabel:`Others`: expense that doesn't fit into any other categories.
+- :guilabel:`Meals`: any form of meal costs related to a project/order.
+- :guilabel:`Gifts`: any form of gift costs related to a project/order.
+- :guilabel:`Mileage`: any form of mileage (gas) costs related to project/order.
+- :guilabel:`Travel & Accommodation`: any travel or accommodation costs related to a project/order.
+
+.. tip::
+   New expense categories can be created from an expense form, by clicking the :guilabel:`Category`
+   field drop-down menu, selecting :guilabel:`View All`, and clicking :guilabel:`New` from the
```

Comment:
I appreciate you pointing this out, but I just checked a 16.4 runbot, and it has "view all" -- so I think I'm going to leave it as-is, assuming the average user reading this will have the most updated version of Odoo 16. However, if you think it would be smart to change it to adhere to 16.3, instead of 16.4, just let me know and I'll make the necessary change right away.